docs/standard/index.md

```diff
@@ -16,7 +16,7 @@ ms.assetid: bbfe6465-329d-4982-869d-472e7ef85d93
 
 > Check out the ["Getting Started with .NET Core" tutorials](../core/getting-started.md) to learn how to create a simple .NET Core application. It only takes a few minutes to get your first app up and running.
 
-.NET is a general purpose development platform. It can be used for any kind of app type or workload where general purpose solutions are used. It has several key features that are attractive to many developers, including automatic memory management and modern programming languages, that make it easier to efficiently build high-quality applications. .NET enables a high-level programming environment with many convenience features, while providing low-level access to native memory and APIs.
+.NET is a general purpose development platform. It can be used for any kind of app type or workload where general purpose solutions are used. It has several key features that are attractive to many developers, including automatic memory management and modern programming languages, that make it easier to efficiently build high-quality applications. .NET enables a high-level programming environment with many convenient features, while providing low-level access to native memory and APIs.
```

Comment:
I think this is not a typo, it's jargon. As I understand it:

* a convenient feature: feature that is convenient to use
* a convenience feature: feature whole sole purpose is to make some task more convenient